services/src/main/java/org/keycloak/services/DefaultKeycloakTransactionManager.java

```diff
@@ -164,6 +164,17 @@ public void commit() {
         }
     }
 
+    private static void commitWithTracing(KeycloakTransaction tx, TracingProvider tracing) {
+        tracing.trace(tx.getClass(), "commit", span -> {
+            try {
+                tx.commit();
+            } catch (RuntimeException e) {
+                tracing.error(e);
+                throw e;
+            }
```

Comment:
```suggestion
            tx.commit();
```

I don't think we need to catch the exception as it should be done in the `OTelTracingProvider` [trace method](https://github.com/keycloak/keycloak/blob/main/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/tracing/OTelTracingProvider.java#L120).